.gitignore

```diff
@@ -0,0 +1,79 @@
+# env
+.env
+
+# virtualenv
+.venv
+venv
+
+# Packaging
+build
+dist
+*.egg-info
+
+# sqlite DB
+db.sqlite3
+*.db
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+
+# Site media
+data/
+
+# Settings
+local.py
+
+# Unit test / coverage reports
+htmlcov/
+.coverage
+.cache
+.pytest_cache
+nosetests.xml
+coverage.xml
+
+# Vagrant
+.vagrant
+
+# PyCharm IDE
+.idea/
+
+# Mr Developer
+.mr.developer.cfg
+.project
+.pydevproject
+
+# Eclipse
+.settings/
+
+# Rope
+.ropeproject
+
+# Sublime-Text:
+/*.sublime-project
+/*.sublime-workspace
+
+# TextMate 2
+.tm_properties
+
+# VS Code
+.vscode
+
+# Mac Os
+.DS_Store
```

Comment:
This should also be removed, this is not our responsibility to add it. And it is not in the Python.gitignore file :-)

Reply:
Done.